backend/src/controllers/FakeController.ts

```diff
@@ -142,6 +142,11 @@ export class FakeController {
         res.status(200).json(fakeRes);
     }
 
+
+    static registerUser(req: Request, res: Response){
+        //TODO FILL IN
```

Comment:
Looking at some of the other functions, make sure to add checks to prevent duplicate usernames.